code/modules/cargo/packs.dm

```diff
@@ -876,6 +876,25 @@
 	crate_type = /obj/structure/closet/crate/secure/engineering
 	dangerous = TRUE
 
+/datum/supply_pack/engine/heavy_emitter
+	name = "Heavy Emitter Crate"
+	desc = "2 weeks ago this bad boy was mounted on IWS 'Bashahar' over at Lizard Empire. Now it can power your sm for like 7500 bucks or something."
```

Comment:
Might sound better as something like:
"2 weeks ago this bad boy was mounted on the IWS Bashahar over at some backwater Lizard drydock halfway to Tizira. Now it can power your SM for like... 7500 credits? How's that sound?"